tests/test_makepackage.py

```diff
@@ -1,57 +1,92 @@
-import platform
+from __future__ import annotations
 import subprocess
+import platform
 from pathlib import Path
-from typing import Dict, List
 
-import pytest
+
+cmd_commands = tuple[str, Path]
```

Comment:
Since it contains only one command and then we create a list of its objects, please call it `cmd_command`.

Reply:
Oh yeah sure

Reply:
Oh yeah yeah, will make the change.